uint16 count
string[] class_list
uint16[] x_mid_list
uint16[] y_mid_list
uint16[] width_list
uint16[] height_list

